src/prng/isaac64.rs

```diff
@@ -358,31 +344,33 @@ mod test {
     #[test]
     fn test_isaac64_construction() {
         // Test that various construction techniques produce a working RNG.
-        let seed = [98784247809, 33887291965896, 0, 0];
-        let mut rng1 = Isaac64Rng::from_seed(&seed);
+        let seed = [1,0,0,0, 23,0,0,0, 200,1,0,0, 210,30,0,0,
+                    0,0,0,0, 0,0,0,0, 0,0,0,0, 0,0,0,0];
+        let mut rng1 = Isaac64Rng::from_seed(seed);
         assert_eq!(rng1.next_u64(), 14964555543728284049);
 
-        /* FIXME: enable once `from_rng` has landed
         let mut rng2 = Isaac64Rng::from_rng(&mut rng1).unwrap();
         assert_eq!(rng2.next_u64(), 919595328260451758);
-        */
     }
 
     #[test]
     fn test_isaac64_true_values_64() {
-        let seed: &[_] = &[1, 23, 456, 7890, 12345];
-        let mut rng1: Isaac64Rng = SeedableRng::from_seed(seed);
+        let seed = [1,0,0,0, 0,0,0,0, 23,0,0,0, 0,0,0,0,
+                    200,1,0,0, 0,0,0,0, 210,30,0,0, 0,0,0,0];
+        let mut rng1 = Isaac64Rng::from_seed(seed);
         let mut results = [0u64; 10];
         for i in results.iter_mut() { *i = rng1.next_u64(); }
         let expected = [
-            547121783600835980, 14377643087320773276, 17351601304698403469,
-            1238879483818134882, 11952566807690396487, 13970131091560099343,
-            4469761996653280935, 15552757044682284409, 6860251611068737823,
-            13722198873481261842];
+                   15071495833797886820, 7720185633435529318,
+                   10836773366498097981, 5414053799617603544,
+                   12890513357046278984, 17001051845652595546,
+                   9240803642279356310, 12558996012687158051,
+                   14673053937227185542, 1677046725350116783];
```

Comment:
Ah, yes, the old seed is too big. It would be good to rebase with an extra commit changing the old seed and results but nothing else, before this commit.